packages/store/src/utils.js

```diff
@@ -0,0 +1,43 @@
+// @ts-check
+
+const { getPrototypeOf, create, fromEntries } = Object;
+const { ownKeys, apply } = Reflect;
+
+const compareStringifies = (left, right) => {
```

Comment:
`compareStringified` reflects what this is doing better. AIUI, it's comparing the stringified versions of its arguments.

Reply:
Good, thanks. Done.